config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -3203,9 +3203,6 @@ presubmits:
     # This covers most of the code related to dynamic resource allocation.
     # Periodic variant: ci-kind-dra
     run_if_changed: /(dra|dynamicresources|resourceclaim|deviceclass|resourceslice|resourceclaimtemplate|dynamic-resource-allocation|pkg/apis/resource|api/resource)/.*.go
-    # The tests might still be flaky or this job might get triggered accidentally for
-    # an unrelated PR.
-    optional: true
```

Comment:
run_if_changed + optional: false doesn't work very well and IMHO we really should prove this out in release blocking first. doing this will introduce a new `Required` status on every PR even if they don't run it (it will be marked ✅  without running) ... it's not one of the better understood prow features.

If we see frequent regressions managing to merge, caught by the release informing/blocking, then we should make this required and deal with having yet another required status on PRs.